File: src/test/java/HomePageTests.java

import Base.TestUtilities;
import Pages.HomePage;
import org.testng.Assert;
import org.testng.annotations.Test;


class HomePageTests extends TestUtilities {

    @Test
    public void assertContactNumberDisplayed() {
        HomePage homePage = new HomePage(driver, log);
        homePage.openHomePage();

        String expectedPhoneNumber = "0123-456-789";

        Assert.assertEquals(homePage.getPhoneNumberText(), expectedPhoneNumber);
    }

}

File: src/test/java/HomePageTests.kt
import Base.TestUtilities
import Pages.HomePage
import org.openqa.selenium.WebDriver
import org.testng.Assert
import org.testng.annotations.Test

class HomePageTests(driver: WebDriver) : TestUtilities() {

    private val homePageRobot = HomePage(driver)

    @Test
    fun assertPhoneNumberDisplayed() {
        homePageRobot.openHomePage()

        val expectedPhoneNumber = "0123-456-789"

        Assert.assertEquals(homePageRobot.getPhoneNumberText(),expectedPhoneNumber)

    }

}